pkg/middleware/stored_session.go

```diff
@@ -19,11 +19,14 @@ type StoredSessionLoaderOptions struct {
 	// Session storage basckend
 	SessionStore sessionsapi.SessionStore
 
-	// How often should sessions be refreshed
-	RefreshPeriod time.Duration
+	// Whether sessions should be refreshed
+	Refresh bool
+
+	// What percentage of the token expiry period to refresh by
+	RefreshPercent uint8
```

Comment:
Why are we using a uint8 for this? As a user, given an option that is called something percent, would you expect to put `8` or `8%` as the value of the option?

Reply:
I'm not sure what significance "8" or "8%" has in your sentence above. Uint8 is a single byte that holds 0 - 255, which is perfect for storing values from 1 to 100. The user enters an integer representing percentage.

See https://github.com/oauth2-proxy/oauth2-proxy/pull/915/files#diff-293f549c0626e25e2554632b028fa5e4402fc0c84a27bd6b01fb7c4b5ceb05afR32
See https://github.com/oauth2-proxy/oauth2-proxy/pull/915/files#diff-ffa1a237cd7ef9acab05b5d5066d8f8011bcc13118919dd98798448a10c10b8dR16